docs/ui/insight/examples.md

```diff
@@ -1,17 +1,54 @@
 ---
-title: "Manage streams and consumer groups in RedisInsight"
-linkTitle: Streams
-description: Learn how to manage streams and consumer groups in RedisInsight
-weight: 1
+title: "RedisInsight examples"
+linkTitle: Examples
+description: 
+weight: 60
 ---
 
+
+RedisInsight allows you to browse, filter, and visualize key-value Redis data structures. It supports CRUD operation for lists, hashes, strings, sets, sorted sets, and so on. You can take advantage of built-in examples in Workbench to learn about the data structures. We'll guide you through one of those examples here and show you other practical ways to use RedisInsight with your databases.
+
+## Use RedisInsight to manage time series 
+
+Examples you find in Workbench allow you interact with some sample data in a few taps. For this orientation, we will use an example called _Redis for time series_. Navigate to this title, read the introduction, then tap the first button to create five time series for bike shops. 
+
```

Comment:
does it make sense to add preconditions? users need the time series capability, which (if they do not have it) is available in the Redis Stack